### ISSUE TITLE
Cannot use polymorphic relationship

### ISSUE THREAD

**Issue by derekwsgray** [orbitjs/ember-orbit#257]
It is difficult to know how to work the difference between how polymorphic relationships are defined in pure Orbit, versus defining models in Ember-Orbit.

In orbit (actually, not in the docs, but gleaned from the unit tests): 
```javascript
        star: {
          keys: {
            remoteId: {}
          },
          relationships: {
            celestialObjects: {
              type: 'hasMany',
              model: ['planet', 'moon'], // <--- polymorphism supplied via array of acceptable types
              inverse: 'star'
            }
          }
       }
```
vs. Ember-Orbit. One might assume you can do this: 
```javascript
      @hasMany(['planet', 'moon']) celestialObjects;
```
given how thin the ember-wrapper seems to be. Has ember-orbit just not caught up with the polymorphism feature?

For comparison, in Ember-Data I had a sub-folder of `/models/abstract` where I would put parent objects, and specifying types like this: `celestialObjects: hasMany('abstract/celestial-object', { polymorphic: true })`.

**Comment by derekwsgray** [comment]
Seems the `@hasMany('celestialObject')` parent class only gets past model schema loading, but doesn't actually work as a polymorphic relationship.

So it appears ember-orbit just doesn't support the polymorphic feature of vanilla orbit.

In `normalize-record-properties.ts`'s `assignRelationships` function, it is assuming a string and missing the possibility of an array:
```typescript
let relationshipType = relationships[relationship].model as string;
```

**Comment by dgeb** [comment]
Resolved via #262 

Thanks @derekwsgray!
